pkg/cmd/init/utils.go

```diff
@@ -0,0 +1,195 @@
+package init
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	msg "github.com/aziontech/azion-cli/messages/init"
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"go.uber.org/zap"
+)
+
+func yesNoFlagToResponse(info *InitInfo) bool {
```

Comment:
This function does not seem to me to be a utils, it is only used once

Reply:
Where would it go then?